src/config.go

```diff
@@ -138,10 +138,7 @@ func (s *Server) getFunctionsResult(function string, deviceIPAddress string, aut
 			}
 		} else {
 			if errRet := s.validateDeviceAccountData(deviceIPAddress, userName, password); errRet != "" {
-				logrus.WithFields(logrus.Fields{
-					"IP address:port": deviceIPAddress,
-					"Username":        userName,
-				}).Errorf(errRet)
+				logrus.Errorf(errRet)
```

Comment:
More targeted error message would be good here (something similar to other error messages in this PR)

Reply:
The value "errRet" already has more targeted message - it is assigned in validateDeviceAccountData function.